sdk/communication/Azure.Communication.PhoneNumbers/tests/SessionRecords/PhoneNumbersClientLiveTests/GetPurchasedPhoneNumbersUsingKeyCredential.json

```diff
@@ -167,7 +218,8 @@
     }
   ],
   "Variables": {
-    "AZURE_COMMUNICATION_LIVETEST_CONNECTION_STRING": "endpoint=https://smstestapp.communication.azure.com/;accesskey=3z7dN2ZEcANlOejjlHaJUfqdC4SyuzdiHbaMcFR0tNRK37KDZ14HUuhIybbBgt6oSQwUwu5TnF5cYFm22Lh/xg==",
-    "RandomSeed": "136581784"
+    "AZURE_COMMUNICATION_LIVETEST_CONNECTION_STRING": "endpoint=https://live-testing.communication.azure.com/;accesskey=Kg==",
+    "COMMUNICATION_CONNECTION_STRING": "endpoint=https://live-testing.communication.azure.com.127.0.0.1.xip.io:9998/;accesskey=Kg==",
```

Comment:
Why is it using both `AZURE_COMMUNICATION_LIVETEST_CONNECTION_STRING` and `COMMUNICATION_CONNECTION_STRING`?
I am assuming that all the tests in this project would use `AZURE_COMMUNICATION_LIVETEST_CONNECTION_STRING`. Something is fishy!

Reply:
I think it is because of here
https://github.com/Azure/azure-sdk-for-net/pull/19587/files#diff-99b1fe381455016be231c2ab9bc830463053e79a858e6d3b17c29a865092e70aR44
The AccessKey property is being parsed out of the `ConnectionString` variable instead of `LiveTestConnectionString`

Reply:
Fixed.